### PR TITLE
Aadhaar kyc verification 

### PR DESCRIPTION
**Pull request by yogesh-pawar-2023**

### CODE REVIEW

File: src/src/adhar_verification/adhar_verification.service.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            };
+            
+            try {
+              const result = await this.httpService.post(url,data ,config).toPromise();          
+              return result.data;
+          } catch (e) {
+              console.log("verify okyc data error", e.message)
+              return e.message
+          }
+      }
```
Comment (sagarkoshti1990) [inline src/src/adhar_verification/adhar_verification.service.ts:74]: please use response format  
```
{
    success: false,
    status: '',
    message: '',
    data: {},
}
```

File: src/src/adhar_verification/adhar_verification.service.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            };
+            
+            try {
+              const result = await this.httpService.post(url,data ,config).toPromise();          
+              return result.data;
+          } catch (e) {
+              console.log("complete okyc data error", e.message)
+              return e.message
+          }
+      }
```
Comment (sagarkoshti1990) [inline src/src/adhar_verification/adhar_verification.service.ts:95]: please use response format
```
{
    success: false,
    status: '',
    message: '',
    data: {},
}
```

File: src/src/adhar_verification/adhar_verification.service.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+              }
+            };
+            
+            try {
+              const result = await this.httpService.get(url,config).toPromise();          
+              return result.data;
+          } catch (e) {
+            console.log("get okyc data error", e.message)
+              return e.message
+          }
```
Comment (sagarkoshti1990) [inline src/src/adhar_verification/adhar_verification.service.ts:113]: please use response format
```
{
    success: false,
    status: '',
    message: '',
    data: {},
}
```

Comment (yogesh-pawar-2023) [reply src/src/adhar_verification/adhar_verification.service.ts]: updated

File: src/src/adhar_verification/adhar_verification.service.ts (hunk 2 above)
Comment (yogesh-pawar-2023) [reply src/src/adhar_verification/adhar_verification.service.ts]: updated

File: src/src/adhar_verification/adhar_verification.service.ts (hunk 1 above)
Comment (yogesh-pawar-2023) [reply src/src/adhar_verification/adhar_verification.service.ts]: updated response type for success and reject also

**Pull request merged**